package.json

```diff
@@ -109,6 +109,7 @@
     "focus-visible": "^5.2.0",
     "fzy.js": "0.4.1",
     "history": "^5.0.0",
+    "hsluv": "1.0.0",
```

Comment:
Ohh this looks nice. Is it worth adding another dependency that does color functions? We already have `color2k`.

If you think it's worth it I'm on board 👍

Reply:
color2k does not support `hsluv` as far as I am aware.

Reply:
No I don't think it does, but it still supports the kind of color manipulations we're doing here. I'm ok with keeping this dependency though. I couldn't find a bundle size, but I imagine it's tiny.

Reply:
> color2k does not support hsluv as far as I am aware.

Yup. color2k doesn't allow us to convert between hex and hsluv, so we need this new dependency.